src/object.ts

```diff
@@ -18,9 +18,17 @@ namespace Shim {
 		return Object.getOwnPropertyNames(o).filter((key) => !Boolean(key.match(/^@@.+/)));
 	}
 
-	export function keys(o: any): string[] {
+	export function symbolAwareKeys(o: any): string[] {
```

Comment:
Yeah I actually had to rename that because I need to use the actual exported `keys` that is declared down towards the bottom.